src/configure/helper/devOps/azureDevOpsHelper.ts

```diff
@@ -13,7 +13,7 @@ const Layer: string = 'azureDevOpsHelper';
 export class AzureDevOpsHelper {
     private static AzureReposUrl = 'dev.azure.com/';
     private static SSHAzureReposUrl = 'ssh.dev.azure.com:v3/';
-    private static VSOUrl = 'visualstudio.com/';
+    private static VSOUrl = '.visualstudio.com/';
```

Comment:
@hiyadav I have changed the url because converting shh based url to https url will result in invalid url [here](https://github.com/microsoft/azure-pipelines-vscode/pull/254/files#diff-3a5fcd2b8cfa0f9e69e9cb735dca4c95R33). Let me know if this looks good.